changes/test_tasks.py

```diff
@@ -89,6 +120,25 @@ def create_identity_lookup(self, lang="eng_ZA"):
             match_querystring=True,
         )
 
+    def create_identity_lookup_with_timestamp(self, timestamp):
+        responses.add(
+            responses.GET,
+            "http://is/api/v1/identities/test-identity-uuid/",
+            json={
+                "results": [
+                    {
+                        "id": "test_identity-uuid",
+                        "details": {
+                            "lang_code": "eng_ZA",
+                            "timeout_timestamp": timestamp,
+                        },
+                    }
+                ]
```

Comment:
This endpoint doesn't return a list of results, rather, it just returns the specific identity